api/strategies/localStrategy.js

```diff
@@ -1,7 +1,17 @@
 const { Strategy: PassportLocalStrategy } = require('passport-local');
 const User = require('../models/User');
 const { loginSchema, errorsToString } = require('./validators');
-const DebugControl = require('../utils/debug.js');
+const winston = require('winston');
```

Comment:
It's best practice to import the logger from a central place

Reply:
I'm not sure if I've done what you requested, and I'm not even sure how api\utils\LoggingSystem.js works, but from the tests I've conducted, everything is functioning correctly without any new errors, so I believe it should be fine

Reply:
You can just import directly from logger.js no need to import from logging system. That file could probably be deleted